package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vuejs.org",
   "private": true,
   "hexo": {
-    "version": "3.7.0"
+    "version": "3.7.1"
```

Comment:
Is this change necessary for the PR?

Reply:
Oops, I bumped it back down in 28d53af. I don't remember specifically wanting to bump up the version of hexo (I haven't use hexo elsewhere); might be from running `npm install`. Anyways, I'll be sure to commit only the necessary files next time. Thanks!